src/main/java/org/opensearch/neuralsearch/search/util/NeuralSparseTwoPhaseUtil.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.opensearch.neuralsearch.search.util;
+
+import org.apache.lucene.search.BooleanClause;
+import org.apache.lucene.search.BooleanQuery;
+import org.apache.lucene.search.BoostQuery;
+import org.apache.lucene.search.Query;
+import org.opensearch.neuralsearch.query.NeuralSparseQuery;
+import org.opensearch.neuralsearch.query.NeuralSparseTwoPhaseParameters;
+import org.opensearch.search.internal.SearchContext;
+import org.opensearch.search.rescore.QueryRescorer;
+import org.opensearch.search.rescore.RescoreContext;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static java.lang.Float.max;
+import static java.lang.Integer.min;
+import static org.opensearch.index.IndexSettings.MAX_RESCORE_WINDOW_SETTING;
+
+public class NeuralSparseTwoPhaseUtil {
```

Comment:
Format all the classes to have all the public functions at start and then all the protected functions followed by private functions. doing this enhance the maintainability and readability of classes.